Extension/src/main.ts

```diff
@@ -100,6 +100,7 @@ async function makeBinariesExecutable(): Promise<void> {
         const commonBinaries: string[] = [
             "./bin/cpptools",
             "./bin/cpptools-srv",
+            "./bin/cpptools_wordexp",
```

Comment:
Can you use a hyphen in the binary name instead of an underscore for consistency with the other binaries we ship?